src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.xml

```diff
@@ -47,7 +48,6 @@
                     </call>
                 </service>
             </argument>
-            <deprecated>The "%service_id%" service is deprecated since Symfony 2.8 and will be removed in 3.0.</deprecated>
```

Comment:
Even if this was unintentionally deprecated. I'd like to keep it that way, so it can stay removed in 3.0. I'm repeating myself, it doesn't make sense to only define the apc one but none else. This is only because of legacy reasons when APC was allaround and the defacto standard. But today apc is the least common cache I would say.